OMS.Web/AdminController.generated.cs

```diff
@@ -449,14 +449,14 @@ public override System.Web.Mvc.ActionResult CreateRole()
         [NonAction]
         partial void CreateRoleOverride(T4MVC_System_Web_Mvc_ActionResult callInfo, OMS.Core.DTO.Role role);
 
-        [NonAction]
-        public override System.Web.Mvc.ActionResult CreateRole(OMS.Core.DTO.Role role)
-        {
-            var callInfo = new T4MVC_System_Web_Mvc_ActionResult(Area, Name, ActionNames.CreateRole);
-            ModelUnbinderHelpers.AddRouteValues(callInfo.RouteValueDictionary, "role", role);
-            CreateRoleOverride(callInfo, role);
-            return callInfo;
-        }
+       // [NonAction]
+        //public override System.Web.Mvc.ActionResult CreateRole(OMS.Core.DTO.Role role)
+       // {
+        //    var callInfo = new T4MVC_System_Web_Mvc_ActionResult(Area, Name, ActionNames.CreateRole);
+         //   ModelUnbinderHelpers.AddRouteValues(callInfo.RouteValueDictionary, "role", role);
+          //  CreateRoleOverride(callInfo, role);
+           // return callInfo;
+       // }
```

Comment:
why did you commented out this code? this file is a generated file and you shouldn't touching this file and let the T4MVC generate the code